lib/linter.js

```diff
@@ -299,6 +304,9 @@ exports.invoke = async function (cwd, args, text, mtime, callback) {
     clearRequireCache(cwd);
     cache = createCache(cwd, eslint_path_arg);
   }
+  if (!cache) {
+    return;
+  }
```

Comment:
I think this need to invoke the `callback`, no?

Reply:
I'm a bit unsure about this one. This is when we, for whatever reason, fail to create a cache. The main reason would be that we fail to resolve a binary.

I've changed it so the callback is called with the provided text, which likely is the most correct semantically. This is sort of inefficient though since it essentially does the same thing as failing silently.